cibuildwheel/windows.py

```diff
@@ -58,7 +58,7 @@ def get_python_configurations(build_selector):
     return python_configurations
 
 
-def build(project_dir, output_dir, test_command, test_requires, test_extras, before_build, build_verbosity, build_selector, repair_command, environment):
+def build(project_dir, output_dir, test_command, test_requires, test_extras, before_build, build_verbosity, build_selector, repair_command, environment, before_test):
```

Comment:
Ugh, these arguments are becoming a mess.

At any rate, I don't think `before_test` should be alone at the end of the argument list. Can you put it maybe after `test_command ` or so?

Reply:
Agree. But there is question if use `kwargs` or reorder arguments? @joerick ?

Reply:
I'd say just reorder now such that it make +- logical sense. Refactoring this (maybe `**kwargs`, but I don't know what the best solution is) is for another time/PR.